.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-  RUST_VERSION_COV: nightly-2024-06-05
+  RUST_VERSION: 1.84.1
```

Comment:
current rust stable seems to be `1.85.0`; should we use the current latest stable here if possible?